src/main.rs

```diff
@@ -109,17 +115,354 @@ fn list_playlists(path: &PathBuf) {
         .for_each(|row| tree.entry(row.parent_id).or_default().push(row));
 
     print_children_of(&tree, PlaylistTreeNodeId(0), 0);
+
+    Ok(())
 }
 
-fn dump_anlz(path: &PathBuf) {
-    let mut reader = std::fs::File::open(&path).expect("failed to open file");
-    let anlz = ANLZ::read(&mut reader).expect("failed to parse setting file");
+fn list_settings(path: &Path) -> rekordcrate::Result<()> {
+    use rekordcrate::DeviceExport;
+
+    let mut export = DeviceExport::default();
+    export.load(path)?;
+    let settings = export.get_settings();
```

Comment:
IMO constructing a mutable default and then loading from a file doesn't seem very rusty to me. Why not make load a constructor? Also allows for nice chaining.
```suggestion
    let settings = DeviceExport::load(path)?.get_settings();
```